### DISCUSSION TITLE
Z-wave gateway?

### DISCUSSION THREAD

**Discussion by rrozema** [mercenaruss/zigstar_gateways#33] [Ideas]
Would it be possible to provide a LAN Gateway that we can plug a USB z-wave stick into, to provide similar functionality as the zigstar lan gateway but instead of zigbee provide z-wave functionality? f.e. to enable people running HA in a machine that has limited or no access to usb ports (f.e. cloud solutions or virtual machines) or to put the z-wave stick in a location physically away from the home assistant machine, to improve reception and/or reduce interference?
I was trying to do so myself using an esp32, a max3421e and esphome but I haven't been able to get it to work yet.

**Comment by mercenaruss** [comment]
Hi, we can have a discussion on this topic privately, please reach me in Telegram: https://t.me/mercenaruss
